chart/templates/logs-persistent-volume-claim.yaml

```
@@ -29,6 +29,10 @@ metadata:
     {{- with .Values.labels }}
     {{- toYaml . | nindent 4 }}
     {{- end }}
+  annotations:
+    {{- if .Values.logs.persistence.annotations}}
+    {{- toYaml .Values.logs.persistence.annotations | nindent 4 }}
+    {{- end }}
```

If `.Values.logs.persistence.annotations` is not provided or equals to {} or null, the manifest will contains an empty `annotations` block which brake the helm upgrade operation, instead you can use `with` function to conditionally  add the annotations key:
```suggestion
  {{- with .Values.logs.persistence.annotations }}
  annotations:
    {{- toYaml . | nindent 4 }}
  {{- end }}
```

---

Thanks for letting me know this. I was unaware. Fixed this issue